supervisor/resolution/checks/addon_pwned.py

```diff
@@ -38,11 +37,13 @@ async def run_check(self) -> None:
             # check passwords
             for secret in secrets:
                 try:
-                    if not await check_pwned_password(self.sys_websession, secret):
-                        continue
+                    await check_pwned_password(self.sys_websession, secret)
+                    continue
```

Comment:
This now break the logic below

Reply:
We only want that part if `check_pwned_password` raises `PwnedSecret` and then the logic is applied?

Reply:
That part of the logic should probably be moved to the `PwnedSecret` exception to make it clearer

Reply:
I can't follow

Reply:
```
                # Check possible suggestion
                if addon.state == AddonState.STARTED:
                    suggestions = [SuggestionType.EXECUTE_STOP]
                else:
                    suggestions = None
                self.sys_resolution.create_issue(
                    IssueType.PWNED,
                    ContextType.ADDON,
                    reference=addon.slug,
                    suggestions=suggestions,
                )
                break
```
That part from line 50